lib/anyway/loaders/yaml.rb

```diff
@@ -10,17 +10,40 @@ module Anyway
   module Loaders
     class YAML < Base
       def call(config_path:, **_options)
-        base_config = trace!(:yml, path: relative_config_path(config_path).to_s) { load_base_yml(config_path) }
+        rel_config_path = relative_config_path(config_path).to_s
+        config = trace!(:yml, path: rel_config_path) { load_base_yml(config_path) }
+        if environmental?(config)
```

Comment:
Shouldn't we first load YAML and only then check whether it's environmental or not?
That could leave unnecessary trace.

Reply:
Moved all in one trace.